op-node/rollup/derive/l1_block_info.go

```diff
@@ -338,6 +468,15 @@ func L1InfoDeposit(rollupCfg *rollup.Config, sysCfg eth.SystemConfig, seqNumber
 				return nil, fmt.Errorf("failed to marshal Interop l1 block info: %w", err)
 			}
 			data = out
+		} else if isIsthmusButNotFirstBlock(rollupCfg, l2Timestamp) {
```

Comment:
this looks wrong, ecotone will always be active when isthmus is active, so we need to move the is isthmus to the top of the if branching